src/CppUTest/SimpleString.cpp

```diff
@@ -478,6 +478,11 @@ SimpleString HexStringFrom(long value)
     return StringFromFormat("%lx", value);
 }
 
+SimpleString HexStringFrom(int value)
+{
+    return StringFromFormat("%x", value);
+}
+
 SimpleString HexStringFrom(signed char value)
 {
     SimpleString result = StringFromFormat("%x", value);
```

Comment:
yes, seems mismerge happened, git didn't give me a conflict report, it just auto-merge and then mis-merged